lib/src/call/quality_issue.dart

```diff
@@ -0,0 +1,398 @@
+part of voximplant;
+
+/// Represents quality issue levels.
+enum VIQualityIssueLevel {
+  /// The quality issue level to indicate that an issue is not detected or
+  /// is resolved.
+  VIQualityIssueLevelNone,
```

Comment:
it looks more like obj-c style, not dart

it is not required to duplicate the enum name in the enum value

Reply:
done